dashboard/app/views/pd/workshop_mailer/exit_survey.html.haml

```diff
@@ -78,9 +78,9 @@
   %p
     Thank you,
     %br
-    Dave Frye
+    Sam Morris
```

Comment:
Lets make this one
Jess Baker
Professional Learning Product Manager

Reply:
Actually on second thought make it from the workshop organizer so its more evergreen